File: Лаба2/Задание 3 вариант 3/main.kt
fun main() {
    val inputs = readLine()!!.split(" ")
    val n = inputs[0].toInt()

    var count = 0
    for (i in 1..n) {
        val g = inputs[i].toInt()
        var number = g
        while (number > 0) {
            val h = number % 10
            number /= 10
            if (h % 3 == 0) {
                count += h
            }
        }
    }

    println(count)
}

File: Лаба2/Задание 3 вариант 3/main.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine("Enter the number of inputs followed by the values:");
        string[] inputs = Console.ReadLine().Split(' ');
        int n = int.Parse(inputs[0]);

        int count = 0;
        for (int i = 1; i <= n; i++)
        {
            int g = int.Parse(inputs[i]);
            while (g > 0)
            {
                int h = g % 10;
                g /= 10;
                if (h % 3 == 0)
                {
                    count += h;
                }
            }
        }
        Console.WriteLine(count);
    }
}
